### PR TITLE
Fix institution's tests

### PR DESCRIPTION
**Pull request by raonismaneoto**
<p><b>Feature/Bug description:</b>
The variables' names weren't generic and the tests weren't using mocks.
</p>

<p><b>Solution:</b>
Replace the variables' names by generics names and use mocks.
</p>

<p><b>TODO/FIXME:</b> n/a</p>

### CODE REVIEW

File: backend/test/followers_institution_handler_test.py [hunk 1]
```diff
@@ -140,45 +140,45 @@ def tearDown(cls):
...
+    cls.user1.institutions_admin = []
+    cls.user1.notifications = []
+    cls.user1.posts = []
+    cls.user1.put()
+    # new User2
+    cls.user2 = User()
+    cls.user2.name = 'User 2'
+    cls.user2.cpf = '089.675.908-91'
+    cls.user2.email = ['user2@email.com']
+    cls.user2.institutions = []
```
Comment (mayzabeel) [inline backend/test/followers_institution_handler_test.py]: remove essas atruibuições vazias, que não altera em nada o teste :)

File: backend/test/institution_timeline_handler_test.py [hunk 2]
```diff
@@ -91,44 +91,44 @@ def test_get(self, verify_token):
...
-    cls.certbio.followers = [cls.mayza.key]
-    cls.certbio.posts = []
-    cls.certbio.admin = cls.mayza.key
-    cls.certbio.put()
+    # new User1
+    cls.user1 = User()
+    cls.user1.name = 'User 1'
+    cls.user1.cpf = '089.675.908-90'
+    cls.user1.email = ['user1@email.com']
+    cls.user1.institutions = []
```
Comment (mayzabeel) [inline backend/test/institution_timeline_handler_test.py]: aqui também

File: backend/test/remove_institution_handler_test.py [hunk 3]
```diff
@@ -27,66 +27,66 @@ def setUp(cls):
...
-    cls.mayza.add_permission("publish_post", cls.certbio.key.urlsafe())
-    cls.raoni.add_permission("publish_post", cls.certbio.key.urlsafe())
-    cls.raoni.put()
-    cls.mayza.put()
+    # new User1
+    cls.user1 = User()
+    cls.user1.name = 'User 1'
+    cls.user1.cpf = '089.675.908-90'
+    cls.user1.email = ['user1@email.com']
+    cls.user1.institutions = []
```
Comment (mayzabeel) [inline backend/test/remove_institution_handler_test.py]: aqui também

File: backend/test/followers_institution_handler_test.py [hunk 4]
```diff
@@ -20,165 +19,105 @@ def setUp(cls):
...
+        self.assertEquals(len(institution.followers), 1,
+                          "The institution should have a follower")
+        self.assertEquals(len(user.follows), 1,
+                          "The user should follow institution")
+
+    @patch('utils.verify_token', return_value={'email': 'user@email.com'})
+    def test_delete(self, verify_token):
+        """Test the institution_follower_handler delete method."""
+        user = mocks.create_user()
+        user.email = ['user@email.com']
```
Comment (andreldsa) [inline backend/test/followers_institution_handler_test.py]: We can improve the user creation by adding an optional argument "email", so we don't to set the email user after creating it.

File: backend/test/followers_institution_handler_test.py [hunk 5]
```diff
@@ -20,165 +19,105 @@ def setUp(cls):
...
+        institution = mocks.create_institution()
+        user.follows = [institution.key]
```
Comment (andreldsa) [inline backend/test/followers_institution_handler_test.py]: Use user.follow()

File: backend/test/followers_institution_handler_test.py [hunk 6]
```diff
@@ -20,165 +19,105 @@ def setUp(cls):
...
+        institution.followers = [user.key]
```
Comment (andreldsa) [inline backend/test/followers_institution_handler_test.py]: Use institution.follow()

**Pull request merged**